plugins/agent-utilities/skills/fusion-parametric-design/tests/test_pipeline_seams.py

```
@@ -721,6 +721,183 @@ def test_a_torn_dump_stops_at_the_orientation_gate_before_any_verdict(self) -> N
+        record = seg.fit_regions(dump, ts.spec())
+        digest = _manifest_hash(build_manifest())
+        without = build_reconstruction_program(
+            parse_fit_record(record), fxr.spec(), manifest_sha256=digest
+        )
+        with_slabs = build_reconstruction_program(
+            parse_fit_record(record), fxr.spec(slabs=True), manifest_sha256=digest, dump=dump
+        )
+        self.assertEqual(1, len([g for g in with_slabs["archetypes"] if g.get("slab")]))
+        self.assertEqual(without["order"], with_slabs["order"])
+        self.assertEqual(without["user_parameters"], with_slabs["user_parameters"])
+        for before, after in zip(without["archetypes"], with_slabs["archetypes"]):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)